### PR TITLE
main/xscreensaver: upgrade to 5.42

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

File: main/xscreensaver/APKBUILD [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 # Maintainer: William Pitcock <nenolod@dereferenced.org>
 pkgname=xscreensaver
-pkgver=5.40
+pkgver=5.42
 pkgrel=0
-pkgdesc="screensavers for X11 environment"
+pkgdesc="Screensavers for X11 environment"
 url="https://www.jwz.org/xscreensaver/"
+arch="all"
 license="GPL"
```
Comment (maxice8) [inline main/xscreensaver/APKBUILD]: license is not SPDX identifier and is ambiguous

**Comment by ncopa** [comment]
This package is also orphaned. @TBK would you like to adopt it?

**Comment by ncopa** [comment]
This can also be moved to community

**Comment by TBK** [comment]
I will update it later fixing the license and moving it to community.

I am not interested in adopting it since I am not currently using it.

**Comment by TBK** [comment]
@ncopa I the changes have been made.

**Comment by ncopa** [comment]
ac6d1869294ed33dac2301d7cc773fde78c61663 e20ae53c740577e721832d6829cb6cdcece0ca59 be952dcfac1911b5685ad02e83d3191c67a62fdf

**Pull request closed without merging**